js/components/ListHeading.react.js

```diff
@@ -1,17 +1,15 @@
-var ListHeading = React.createClass({
-  render: function() {
+import React from "react";
+import { Link } from "react-router";
+
+const ListHeading = ({selectTag, text}) => {
```

Comment:
Is the selectTag still necessary here? I can't see another reference in the code to it.